aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -48,6 +48,10 @@ impl MoveVmExt {
 
         let treat_friend_as_private = features.is_enabled(FeatureFlag::TREAT_FRIEND_AS_PRIVATE);
 
+        let disable_invariant_violation_check = false; // TODO: timed_features.is_enabled(TimedFeatureFlag::DisableInvariantViolationCheckInSwapLoc);
+
```

Comment:
@vgao1996 @runtian-zhou I think we want to keep it to false for now, right?

Reply:
We can keep it to false for now, since the timed feature hasn't been enabled in testnet/mainnet, but IMO a safer way to handle this is just to wire up the timed features now, so that we don't forget it in the future.

Reply:
Ok, I added the timed features. But notice  the current state at main for the timed features is very old (everything unspecified, no feature on).